src/objects.js

```diff
@@ -1,41 +1,55 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name,
+    age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
```

Comment:
in your code editor you'll notice that `.hasOwnProperty` has been underlined in red. If you hover your mouse over it will tell you the reason why. In this case, it's because the linter/formatter had discouraged against using `.hasOwnProperty`. An alternative way to check if an object has a property is to use the [`in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) operator. e.g.
```
return property in object;
```